src/screens/Auth/CredentialInputScreen.js

```diff
@@ -16,15 +16,15 @@ const CredentialInputScreen = props => {
         titleText: 'Sign in',
         changeFormText: 'Don\'t have an account yet? Register',
         otherFormType: formTypes.SIGN_UP,
-        googleButtonText: 'Sign in with Google',
+        googleButtonText: 'Continue with Google',
       };
       break;
     case formTypes.SIGN_UP:
       options = {
         titleText: 'Sign Up for Civic',
```

Comment:
For consistency (and to match the mocks), do you mind changing to a lowercase u in "up"?